index.html

```diff
@@ -76,6 +129,22 @@ <h2 id=intro>Introduction</h2>
 by the internationalization Working Group
 for more information.)
 
+    <aside class=note title="Ruby as notes">
+      Notes are sometimes placed between lines
+      similarly to how ruby is laid out
+      (inter-linear notes).
+      <span class="checkme">
+      The processing of this arrangement is not covered in this document.
+      Putting line break within ruby string is not allowed for mono-ruby. 
+      In addition, putting line break within ruby base text is not allowed 
+      for group-ruby. 
+      In contrast, interlinear notes may have large number of characters, 
+      putting line breaks within both ruby base text and ruby string
+      need to be allowed where appropriate, and that lead the processing 
+      to be different from ruby considered in this note. 
```

Comment:
Perhaps change "in this note" to "in this document"? (It is not clear the "note" refers to this "Ruby as notes" note, or the whole document, which will be an i18n WG Note in the future.)

Reply:
ah, yes. I've put 'note' as this document is WG Note,,, but should be better to use "document".